Enhance the Zowe CLI sample plug-in to provide an SCRT test tool (#147)
zowe/zowe-cli-sample-plugin: docs/tutorials/customize_your_config_file/Readme.md

> The zowe.config_sample.json file in this folder can be customized to enable you to run each of the available commands of the zowe sample plug-in.

Reviewer: ```suggestion
The `zowe.config_sample.json` file in this folder can be customized to enable you to run each of the available commands of the Zowe sample plug-in.
```

> 1. Install Zowe CLI.  See instructions in the sample plugin's main [README](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/README.md) file.

Reviewer: ```suggestion
1. Install Zowe CLI.  See instructions in the sample plug-in's main [README](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/README.md) file.
```

> 2. Install the Zowe sample plug-in.  Again, see instructions in the sample plugin's main [README](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/README.md)) file.

Reviewer: ```suggestion
2. Install the Zowe sample plug-in.  Again, see instructions in the sample plug-in's main [README](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/README.md)) file.
```

> 3. Copy zowe.config_sample.json to zowe.config.json in a directory of your choice.

Reviewer: ```suggestion
3. Copy `zowe.config_sample.json` to `zowe.config.json` in a directory of your choice.
```

> 4. Update the properties in zowe.config.json to reflect the values used at your company.

Reviewer: ```suggestion
4. Update the properties in `zowe.config.json` to reflect the values used at your organization.
```

5. You can then successfully run each of the following sample plug-in commands:

```
zowe zowe-cli-sample fail error-handler
zowe zowe-cli-sample issue command-with-options
zowe zowe-cli-sample issue command-with-options --required-number 1111
zowe zowe-cli-sample issue command-with-options --required-number 1111 --use-the-string

> zowe zcsp issue command-with-options --required-number 1111 --use-the-string --the-string "Test string"

Reviewer 2: why only use the alias here? 😋

Author: The only reason that I used the alias was that the command was so long that it wrapped in some markdown displays (like the way that your comment above wrapped).
